ci/Jenkinsfile

```diff
@@ -98,8 +100,8 @@ pipeline {
         additionalOptions: '-dpi 1'
       ]) { script {
         sh "fluxbox &"
-        env.TESTRAIL_USR = "${CREDS_USR}"
-        env.TESTRAIL_PWD = "${CREDS_PSW}"
+        env.TESTRAIL_USR = "${TESTRAIL_CREDS_USR}"
+        env.TESTRAIL_PWD = "${TESTRAIL_CREDS_PSW}"
```

Comment:
This should be acquired in this stage, not in the global `environments` section.